### DISCUSSION TITLE
How to submit an answer developer with OOP

### DISCUSSION THREAD

**Discussion by jenvigo** [codewars/codewars.com#2474] [Q&A]
After solving a really hard kata, I have just noticed that I need to translate all my classes, interfaces, etc.. into plain PHP spaguetti code to submit the answer. Is there a way to avoid this?  Thanks in advance.

**Comment by DonaldKellett** [comment]
If the kata requires you to submit a function, you have to submit a function. What you _could_ do is implement all your logic in classes, but then just create an instance of the class and invoke an appropriate method in the function body.

**Reply by jenvigo** [reply]
Thanks, I will try that way.
